.github/workflows/build.yml

```diff
@@ -109,3 +109,59 @@ jobs:
     
     - run:  bundle exec rake brakeman
       name: Brakeman
+
+  test:
+    name: Tests
+    needs: [build]
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        tests: [unit, integration]
+        feature-flags: [on, off]
+        include:
+          - tests: unit
+            include-pattern: spec/**/**/*_spec.rb
+            exclude-pattern: spec/{system,requests}/**/*_spec.rb
+          - tests: integration
+            include-pattern: spec/{system,requests}/**/*_spec.rb            
+            exclude-pattern: spec/**/**/*_spec.rb
+    services:
+      redis: 
+        image: redis:alpine
+        ports:
+          - 6379:6379
+      postgres:
+        image: postgres:11.10
```

Comment:
Oh, I hadn't noticed this version bump in previous PR — will migration be ok? think we're stuck on 9.6 in Azure?

Reply:
yes will be fine...TTAPI was also on 9.6 on Azure..moved to 11 on PaaS